src/app/practice/back-line/page.tsx

```diff
@@ -1,8 +1,9 @@
-import { Exercises } from "@/components/Exercises";
+import { Exercises } from '@/components/Exercises'
 
 const exerciseData = {
   title: 'Back Line Conditioning',
-  description: 'By targeting the back line fascia, these exercises promote flexibility, strength, and overall health.',
+  description:
```

Comment:
Your IDE seems to be applying a strange formatting to strings throughout the project.